src/app/mailviewer/singlemailviewer.component.ts

```diff
@@ -164,19 +166,27 @@ export class SingleMailViewerComponent implements OnInit, DoCheck, AfterViewInit
   public ngOnInit() {
     this.messageActionsHandler.mailViewerComponent = this;
     this.showHTMLDecision = localStorage.getItem(showHtmlDecisionKey);
+    // Update 2020-12, now preferring resizerPercentageKey
     this.previousHeight = parseInt(localStorage.getItem(resizerHeightKey), 10);
+    this.previousHeightPercentage = parseInt(localStorage.getItem(resizerPercentageKey), 10);
   }
 
   public ngAfterViewInit() {
     // These two viewchildren queries are needed for loading an email from
     // a URL fragment:
 
     // Ensure resizer child is loaded before setting height
-    this.resizerQuery.changes.subscribe((resizer: HorizResizerDirective) => {
+    this.resizerQuery.changes.subscribe((resizer: QueryList<HorizResizerDirective>) => {
       setTimeout(() => {
         if (this.adjustableHeight) {
           if (this.previousHeight) {
-            this.resizer.resizePixels(this.previousHeight);
+            // upgrade to Percentage
+            this.previousHeightPercentage = resizer.first.heightOffsetToPercentage(this.previousHeight);
+            this.previousHeight = undefined;
+            localStorage.setItem(resizerHeightKey, '');
```

Comment:
Any reason not to go all-in with `removeItem()`?

Reply:
Not other than laziness in me looking up what that was called ;)